lmp/bb-config.sh

```diff
@@ -195,14 +195,17 @@ fi
 
 # Add build id H_BUILD to output files names
 if [ "$CONF_VERSION" == "1" ]; then
-	cat << EOFEOF >> conf/local.conf
-DISTRO_VERSION_append = "-\${H_BUILD}-\${LMP_VERSION}"
+	cat << 'EOFEOF' >> conf/local.conf
+DISTRO_VERSION_append = "${DISTRO_VERSION_EXTENDED}"
 EOFEOF
 else
-	cat << EOFEOF >> conf/local.conf
-DISTRO_VERSION:append = "-\${H_BUILD}-\${LMP_VERSION}"
+	cat << 'EOFEOF' >> conf/local.conf
+DISTRO_VERSION:append = "${DISTRO_VERSION_EXTENDED}"
 EOFEOF
 fi
+cat << 'EOFEOF' >> conf/local.conf
+DISTRO_VERSION_EXTENDED ?= "-${H_BUILD}-${LMP_VERSION}"
```

Comment:
Just to facilitate the reading process here, can we first define this variable and later used it on the code blocks changed above?

Reply:
I will do